provision/ubuntu/install.sh

```diff
@@ -38,7 +35,28 @@ sudo apt-get install -y --allow-downgrades \
     libtool cmake realpath m4 automake \
     protobuf-compiler libprotobuf-dev libyaml-cpp-dev \
     socat pv tmux bc gcc-multilib binutils-dev \
-    binutils wget rsync ifupdown
+    binutils wget rsync ifupdown \
+    python-sphinx python-pip \
+    libncurses5-dev libslang2-dev gettext \
+    libselinux1-dev debhelper lsb-release \
+    po-debconf autoconf autopoint
+
+# Install nsenter for kubernetes
+cd /tmp && \
```

Comment:
Could we use a variable for the version? 

On the  other hand, could you delete all `&& \` for this installation. All other scripts are not following this format

Reply:
But the CLANG installation is written with the `&& \` format, do you want me to clean that one as well?

Reply:
If you can, it'll be great :-)